composer.json

```diff
@@ -25,6 +25,7 @@
     "pimcore/payment-provider-paypal-smart-payment-button": "^2.0",
     "pimcore/ecommerce-framework-bundle": "^1.0",
     "pimcore/web-to-print-bundle": "^1.0",
+    "pimcore/newsletter-bundle": "^1.0",
```

Comment:
Do we really need to require this here since the CMF already covers this?

Reply:
yes, we should not rely on CMF solely to install this bundle.